### DISCUSSION TITLE
The server appears to be unable to contact the other nodes in the cluster

### DISCUSSION THREAD

**Discussion by nettabaniel1982** [cockroachdb/cockroach#65450] [Q&A]
Hi, I'm new to CockroachDB.
Just installed Core version and started a local cluster with this lab:
https://www.cockroachlabs.com/docs/v21.1/start-a-local-cluster

After the first step (start a cluster) I get this warning:
* WARNING: The server appears to be unable to contact the other nodes in the cluster. Please try:
*
* - starting the other nodes, if you haven't already;
* - double-checking that the '--join' and '--listen'/'--advertise' flags are set up correctly;
* - running the 'cockroach init' command if you are trying to initialize a new cluster.
*
* If problems persist, please see https://www.cockroachlabs.com/docs/v21.1/cluster-setup-troubleshooting.html.

Well... of course.. I didn't start them yet. The instructions tell me to start them one at a time.
What am I missing?

**Comment by jordanlewis** [comment]
Hey @nettabaniel1982, did you try starting up the other nodes too?
